package/rmfakecloud-proxy/rmfakecloudctl

```diff
@@ -0,0 +1,319 @@
+#!/usr/bin/env bash
+# Copyright (c) 2021 ddvk
+# Copyright (c) 2021 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+set -euo pipefail
+
+# Path where the certificate authority key and proxy TLS certificate are stored
+data_dir=/opt/var/rmfakecloud-proxy
+
+# Path where system CA certificates are stored
+local_cert_dir=/usr/local/share/ca-certificates
+
+# Path to the static hosts definition
+hosts_path=/etc/hosts
+
+# Path to the configuration directory
+conf_dir=/opt/etc/rmfakecloud-proxy
+
+# Proxy listen address
+proxy_listen=127.0.42.10
+
+# Path to the Xochitl configuration file
+xochitl_conf_path=/home/root/.config/remarkable/xochitl.conf
+
+# Path to Xochitl data
+xochitl_data_dir=/home/root/.local/share/remarkable/xochitl
+
+# Disconnect Xochitl from the cloud
+#
+# This is a necessary step when switching cloud servers to prevent the client
+# from believing all the files have been deleted.
+disconnect-cloud() {
+    if grep -q "^devicetoken=" "$xochitl_conf_path" \
+        && grep -q "^usertoken=" "$xochitl_conf_path"; then
+        echo "Xochitl sync is enabled. Disabling..."
+    else
+        return 0
+    fi
+
+    # Make sure Xochitl is not running
+    local was_active
+
+    if systemctl --quiet is-active xochitl.service 2> /dev/null; then
+        systemctl stop xochitl
+        was_active=1
+    fi
+
+    if pgrep xochitl > /dev/null; then
+        killall xochitl
+        sleep 1s
```

Comment:
Wouldn't it make more sense to wait a slight bit longer, and to exit early if no xochitl processes exist?

Reply:
Something like the solution proposed there? https://stackoverflow.com/a/41613532

Reply:
Yeah, I would probably use the answer below the one you linked
```bash
while kill -0 PIDS 2> /dev/null; do sleep 0.5; done;
```
And just modify it to have a timeout.